dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -242,6 +247,14 @@ public class ConnectorArguments extends DefaultArguments {
           .ofType(Integer.class)
           .describedAs("N");
 
+  private final OptionSpec<ChronoUnit> optionQueryLogInterval =
+      parser
+          .accepts(OPT_QUERY_LOG_INTERVAL, "The interval between each query log dump")
+          .withOptionalArg()
+          .ofType(ChronoUnit.class)
+          .withValuesConvertedBy(ChronoUnitValueConverter.INSTANCE)
```

Comment:
`HOURS` and `DAYS` are good internal constants, but I think they should not be part of the command-line options. I would like to be able to provide the values in format `1h` or `1d` (similarly `1w`, `1m` and `1y` but these bigger units are not necessary rigth now - we can add them later), so that it could be easily extended in the future.

Another option would be to rename the field to `--query-log-rotation-frequency` and use `hourly` and `daily` as values.